content/english/java-basics/activity-1.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Print Statements and Comments"
+description: "Introduce println() methods and comments in Java."
+date: 2020-07-08T00:00:00Z
```

Comment:
Please add weights to all the pages (excluding _index.md) to denote the order of the pages (e.g. weight: 1 will be the first page after the main page, etc.)